packages/node/test/unit/apps.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Node } from "../../src/node";
+
+describe("Apps", () => {
+  let node: Node;
+  beforeEach(() => {
+    node = new Node();
+  });
+
+  it("can open a connection with an app", () => {
+    const appInstanceEventEmitter = node.openApp("1");
+    expect(appInstanceEventEmitter).toBeDefined();
```

Comment:
Is this really the best test of a connection? Can't you send a test message?

Reply:
This test has a poor description. No listeners exist for this event emitter to be useful at the moment.

When they do, appropriate tests for them will be added. This should just be confirming that an EventEmitter is given back.

Reply:
give it a better description?

Reply:
~can open a connection with an app~